Showcase-iOS/Contact Us/View/ContactUsCollectionViewCell.swift

```diff
@@ -66,31 +61,37 @@ class ContactUsCollectionViewCell: UICollectionViewCell {
         }
     }
     
-    private func populateStaticMap(){
-        let temp = "\(staticMapbaseUrl)\(viewModel.latitude),\(viewModel.longitude)"
-        guard let mapUrl = URL(string: temp) else {
-            mapImage.image = #imageLiteral(resourceName: "placeHolder")
-            mapImage.contentMode = .scaleAspectFit
-            return
-        }
-        mapImage.kf.setImage(with: mapUrl)
+    private func setupMap() {
+        let location = createLocation()
+        let region = createRegion(location: location)
+        let pin = createPin(region: region)
+        map.setRegion(region, animated: false)
+        map.addAnnotation(pin)
     }
     
-    private func styleView() {
-        email.setTitleColor(UIColor.DvtBlueColor, for: .normal)
-        call.setTitleColor(UIColor.DvtBlueColor, for: .normal)
-        navigate.setTitleColor(UIColor.DvtBlueColor, for: .normal)
-        
+    private func createLocation() -> CLLocationCoordinate2D {
+        let longitude = CLLocationDegrees(exactly: viewModel.longitude) ?? 0
+        let latitude = CLLocationDegrees(exactly: viewModel.latitude) ?? 0
+        let location = CLLocationCoordinate2D(latitude: latitude,
+                                              longitude: longitude)
+        return location
     }
     
-    private func addTapRecogniserToStaticImage() {
-        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(staticMapImageTapped(tapGestureRecognizer:)))
-        mapImage.isUserInteractionEnabled = true
-        mapImage.addGestureRecognizer(tapGestureRecognizer)
+    private func createRegion(location: CLLocationCoordinate2D) -> MKCoordinateRegion {
+        let span = MKCoordinateSpan(latitudeDelta: 0.01, longitudeDelta: 0.01)
```

Comment:
Can we also zoom out a little so that we can see the some of the surrounding roads.
```suggestion
        let span = MKCoordinateSpan(latitudeDelta: 0.01, longitudeDelta: 0.01)
```

Reply:
<img width="495" alt="Screenshot 2019-03-11 at 09 06 48" src="https://user-images.githubusercontent.com/22531825/54105900-2212c680-43dd-11e9-986c-e3dd667284a7.png">

Is this okay?

Reply:
seems better. Can see some road names =)